### PR TITLE
chore(angular): Update to most recent version of angular. #227

### PR DESCRIPTION
**Pull request by hoganmaps**

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -2,15 +2,15 @@
   "name": "ozpwebtop",
   "version": "0.4.16",
   "devDependencies": {
-    "angular-mocks": "~1.2",
+    "angular-mocks": "1.4.1",
```
Comment (arw180) [inline bower.json]: Is there a reason you explicitly lock down version numbers here? I would think we'd at least want to allow for patch version updates (`~1.4.1`, for example). (this applies to all of the versions listed here, as it looks like all of the updates specify a specific version number)

Comment (hoganmaps) [inline bower.json]: That is a good idea, I think I was thinking lock it in to a specific version in the bower resolutions, but i will add the tilde to all the changed deps

Comment (arw180) [inline bower.json]: Agreed, that sounds good

**Comment by SOAP1914** [comment]
Looks good to me nothing major change

**Pull request merged**